@here/olp-sdk-authentication/lib/UserAuth.ts

```diff
@@ -241,7 +241,8 @@ export class UserAuth {
 
         const headers = new Headers({
             Authorization: "Bearer " + token,
-            "Content-Type": "application/json"
+            "Content-Type": "application/json",
+            "User-Agent": "OLP-TS-SDK/1.x.x"
```

Comment:
The 1.x.x should actually be the version of the SDK, e.g. 1.3.0 or 1.3.1.
It definitely should not be "1.x.x".

Reply:
Updated